### PR TITLE
Task #40275 Supervisor edit course info

### PR DESCRIPTION
**Pull request by ManhLeDuc**
## Related Tickets
- [#40275](https://edu-redmine.sun-asterisk.vn/issues/40275)

## WHAT (optional)
- Change Course Info for Supervisor

## HOW
- Ensure user is Supervisor role
- Ensure user is Supervisor of edited course

## Evidence (Screenshot or Video)
image

image

### CODE REVIEW

**Comment by trannguyen61** [comment]
OK

**Comment by ManhLeDuc** [comment]
ready

**Comment by nguyenquanganhc** [comment]
OK

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+  <div class="col-3">
+    <%= image_tag Settings.pic_url, alt: @course.name, class: "card-img-top" %>
+  </div>
+  <div class="col-9 d-flex flex-column bd-highlight mb-3">
+    <h2 class="p-2 bd-highlight">
+      <span class="badge bg-info text-dark"><%= @course.status %></span>
+      <%= @course.name %>
+    </h2>
+    <div class="p-2 bd-highlight">
+      <%= "#{t ".expected_time"}: #{@course.start_time.strftime "%d/%m/%Y"}-#{@course.estimated_end_time.strftime "%d/%m/%Y"}" %>
```
Comment (ngocvt-0484) [inline app/views/courses/show.html.erb]: cho "%d/%m/%Y" vô setting

**Comment by ManhLeDuc** [comment]
ready

**Pull request merged**